src/components/PDFThumbnail/index.tsx

```diff
@@ -0,0 +1,51 @@
+// @ts-expect-error - We use the same method as PDFView to import the worker
+import pdfWorkerSource from 'pdfjs-dist/legacy/build/pdf.worker';
```

Comment:
Can you please the error instead of ignoring it? And if an error is expected the comment should explain the reason

Reply:
Added comment to indicate that the legacy worker js lacks TypeScript typings.